src/main/java/com/amazon/pages/AbstractPage.java

```diff
@@ -0,0 +1,55 @@
+package com.amazon.pages;
+
+import com.amazon.services.GoTo;
+import org.apache.log4j.Logger;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.remote.RemoteWebDriver;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+
+public class AbstractPage {
+
+    private static final Logger LOGGER = Logger.getLogger(AbstractPage.class);
+
+    //    WebDriver driver;
```

Comment:
Please if you comment something in your code, add additional comment, why it needed and not removed. Or remove useless code

Reply:
Description added